pyquil/api/_compiler.py

```diff
@@ -356,6 +356,23 @@ def reset(self) -> None:
         """
         self._qpu_compiler_client = None
 
+    @property
+    def client(self) -> Client:
+        """Return the `Client` for the compiler (i.e. quilc, not translation service)."""
+        # TODO(notmgsk): This was introduced around 2.25 to provide
```

Comment:
This should be in the docstring, and not version-specific. While that seems like important information within the context of the PR, it becomes irrelevant as more information comes out.

Also, this might be worth having as an `@abstractmethod` on the `AbstractCompiler` with the documentation note there instead. That's the intent behind abstract methods in any case - parity among subclasses.

Reply:
I removed the comment entirely, and have chosen to not take the abstract method route just because at this point I'm not sure how much it gives us, but if you feel strongly about it I will add it.

Reply:
well, now that the methods are identical, I think it's wise to make it a non-abstract method on the parent class for de-duplication